designs/tripleo/osc-tripleo.md

```diff
@@ -73,6 +73,20 @@ description: >
     description: image
     type: string
 	default: "/var/lib/osc/data"
+		
+  # The parameter contains the path where OSC certificate file resides.
+  OSCTrustStoreFile:
+    default: '/osc-cert/osctruststore.jks' 
```

Comment:
can we just place these files in the same source volume? Basically i would look to see how we can use the 1 persistent volume to contain all the persistent data.

In case of failover etc we can mount a single volume and continue operating vs the current mode which seems to indicate we need to provide each of these jks and pk12 files seperately.

Reply:
Yes that is the intention, the location specified is the source location on the director node. The same is copied on the overcloud controller node at the same location. During the Docker Run, we copy these certificates to destination volume, for example in /opt/vmidc/bin/data. I will add this as a note so that we can discuss in the upcoming reviews to see any potential conflicts/issues.